src/screens/EmptyScreen.tsx

```diff
@@ -32,6 +38,18 @@ const EmptyScreen: React.FC<ScreenProps> = ({ navigation }) => {
     if (savedTech) console.warn(savedTech);
   };
 
+  const adicionaAtividade = async () => {
```

Comment:
só pra manter o padrão de nomenclatura

```suggestion
  const addActivity = async () => {
```